.github/workflows/deploy_production.yml

```
@@ -0,0 +1,55 @@
+name: Deploy production app to Firebase App Distribution
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy_android:
+    name: Deploy Android app to Firebase App Distribution
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    environment: production
+    defaults:
+      run:
+        working-directory: ./sample
+    steps:
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - name: Checkout source code
+        uses: actions/checkout@v3
+
+      - name: Cache Gradle
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.gradle/caches/modules-*
+            ~/.gradle/caches/jars-*
+            ~/.gradle/caches/build-cache-*
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Decode singing properties
+        run: echo "${{ secrets.ANDROID_SIGNING_PROPERTIES }}" | base64 --decode > signing.properties
+
+      - name: Decode release keystore
+        run: echo "${{ secrets.ANDROID_RELEASE_KEYSTORE }}" | base64 --decode > config/release.keystore
+
+      - name: Decode buildKonfig properties
+        run: echo "${{ secrets.BUILD_KONFIG_PROPERTIES }}" | base64 --decode > buildKonfig.properties
```

- I think we can provide these properties files without base64 encoding + combine all in 1 step. That means, we can put raw value in GH secrets for `BUILD_KONFIG_PROPERTIES` and `ANDROID_SIGNING_PROPERTIES` because they're text files. We still need base64 for `ANDROID_RELEASE_KEYSTORE` because it's a binary file 🤓 
- Please help to migrate [the current usage of `BUILD_KONFIG_PROPERTIES`](https://github.com/nimblehq/kmm-templates/blob/71cbe983c811625de8d9dae39b2a5c6653ed7266/.github/workflows/test.yml#L104) to non-base64 encoding too 🙏 

```suggestion
      - name: Set up necessary config
        run: |
          echo "${{ secrets.BUILD_KONFIG_PROPERTIES }}" > buildKonfig.properties
          echo "${{ secrets.ANDROID_SIGNING_PROPERTIES }}" > signing.properties
          echo "${{ secrets.ANDROID_RELEASE_KEYSTORE }}" | base64 --decode > config/release.keystore
```